gtk4/src/auto/editable.rs

```diff
@@ -24,6 +27,11 @@ impl Editable {
 }
 
 pub trait EditableExt: 'static {
+    #[cfg(any(feature = "v4_10", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v4_10")))]
+    #[doc(alias = "gtk_editable_delegate_get_accessible_platform_state")]
+    fn delegate_get_accessible_platform_state(&self, state: AccessiblePlatformState) -> bool;
```

Comment:
this looks like a wrongly annotated API

Reply:
No, that's actually correct according to the C API.